frontend/components/MainPage/sections/OlasBalanceSection.tsx

```diff
@@ -177,21 +127,45 @@ export const MainOlasBalance = () => {
   }, [totalOlasBalance]);
 
   return (
-    <CardSection vertical gap={8} bordertop="true" borderbottom="true">
+    <CardSection
+      vertical
+      gap={8}
+      bordertop="true"
+      borderbottom="true"
+      padding="16px 24px"
+    >
       {canShowAvoidSuspensionAlert ? <AvoidSuspensionAlert /> : null}
       <LowTradingBalanceAlert />
+
       {isBalanceLoaded ? (
-        <>
-          <CurrentBalance />
+        <Flex vertical gap={8}>
+          <Text type="secondary">Current balance</Text>
           <Flex align="end">
             <span className="balance-symbol">{UNICODE_SYMBOLS.OLAS}</span>
             <Balance className="balance">{balance}</Balance>
             <span className="balance-currency">OLAS</span>
           </Flex>
-        </>
+
+          <Text
+            type="secondary"
+            className="text-sm pointer hover-underline"
+            onClick={() => setIsAccountBalanceDetailsModalVisible(true)}
+          >
+            See breakdown
+            <RightOutlined style={{ fontSize: 12, paddingLeft: 6 }} />
+          </Text>
+        </Flex>
       ) : (
         <Skeleton.Input active size="large" style={{ margin: '4px 0' }} />
       )}
+
+      {isAccountBalanceDetailsModalVisible && (
+        <AccountBalances
+          hideAccountBalanceDetailsModal={() =>
```

Comment:
wonder if it's better to use `useCallback` here since all the data re-requests trigger re-render every 5sec